packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable no-console */
```

Comment:
@mitra-deriv Do we really need this line? I'm asking because I don't see any console here) The rest is looking good for me